### ISSUE TITLE
[FireMonkey] After latest Nightly, CSS' scripts put tabs on endless loading situation

### ISSUE THREAD

**Issue by EstherMoellman** [erosman/support#183]
Hi @erosman ! How are you?

As title says, today after latest Nightly' update, CSS' scripts in FireMonkey put tabs on endless loading situation... tabs can't finish loading.

If I have a global CSS' script, then all the tabs only load when I disable this global CSS' script.
The same happens with CSS' script for specific domains, they only work when I disable their specific CSS' scripts.

I'm 100% sure that there is no problem with the scripts, they were working fine for ages.
The issue only appeared today, after Nightly' update.
But the issue seems not to affect JS' scripts.

Perhaps the next Nightly' update will fix the issue, but I preferred to alert you about the current situation.

Big hug! : )

**Comment by erosman** [comment]
Hi Esther

I am also on Nightly but I have not seen any issues.

I cant think of any reason why a content CSS could result in a tab being reloaded since there is no process to reload a tab.

Let's see what happens when Nightly is updated again.

**Comment by EstherMoellman** [comment]
... the issue seems to be solved... but I'm not sure about the culprit, nor about what solved the issue.

Yesterday after opening here the issue, I made some "detective job", and I found that deleting my policies.json file solved the issue. However, doing some more detective job, I also found that cleaning my startupCache folder solved the issue too. But all this detective job happened in the middle of a new Nightly' update, and I'm not really sure about what solved the issue. Anyway, seems to be solved : )
